rotkehlchen/chain/evm/decoding/decoder.py

```diff
@@ -62,16 +62,16 @@ def __call__(
             decoded_events: list['EvmEvent'],
             action_items: list[ActionItem],
             all_logs: list[EvmTxReceiptLog],
-    ) -> tuple[Optional['EvmEvent'], list[ActionItem]]:
+    ) -> DecodingOutput:
         ...
 
 
-@dataclass(init=True, repr=True, eq=True, order=False, unsafe_hash=False, frozen=True)
+@dataclass(init=True, repr=True, eq=True, order=False, unsafe_hash=False, frozen=False)
```

Comment:
why make it mutable?

Reply:
Too, prev implementation. Will undo this now